daemon/containerd/service.go

```diff
@@ -74,6 +78,14 @@ func NewService(config ImageServiceConfig) *ImageService {
 	}
 }
 
+func (i *ImageService) snapshotterService(snapshotter string) snapshots.Snapshotter {
+	if s, ok := i.snapshotterServices[snapshotter]; ok {
+		return s
+	}
+
+	return i.client.SnapshotService(snapshotter)
```

Comment:
Should this be stored in the map, or was it intentional to only keep a reference to the default one?